src/register.js

```diff
@@ -141,10 +143,10 @@ export function wrapDidDocument(did, owner, history) {
     } else {
       if (
         delegateCount > 0 &&
-        (event._eventName === 'DIDDelegateChanged' ||
-          (event._eventName === 'DIDAttributeChanged' &&
-            bytes32toString(event.name).match(/^did\/pub\//))) &&
-        validTo.lt(now)
+    (event._eventName === 'DIDDelegateChanged' ||
```

Comment:
Statements that span multiple lines being indented less than the starting line is confusing and inconsistent with the original code style.  Can we revert these changes since they are unnecessary and reduce readability?  This happens in a few other places as well (line 19-22, line 62).

Reply:
I didn't touch this part. Is from the previous code already. Should I fix it anyways?

Reply:
It is showing as a change in the diff